config.schema.json

```
@@ -36,6 +36,11 @@
         "name": "Device Discovery",
         "type": "boolean",
         "description": "Enable Device Info Logging."
+      },
+      "appendDeviceTypeSuffix": {
+        "name": "Device Type Suffix",
+        "type": "boolean",
+        "description": "Append device type onto device name on discovery."
```

As an option to prevent the breaking change of removing suffixes from people that have already upgraded you might consider including the default.

```suggestion
        "description": "Append device type onto device name on discovery.",
        "default": true
```